akka-docs/rst/scala/stream/stream-quickstart.rst

```diff
@@ -248,7 +248,7 @@ As you can see, inside the :class:`GraphDSL` we use an implicit graph builder ``
 using the ``~>`` "edge operator" (also read as "connect" or "via" or "to"). The operator is provided implicitly
 by importing ``GraphDSL.Implicits._``.
 
-``GraphDSL.create`` returns a :class:`Graph`, in this example a :class:`Graph[ClosedShape, Unit]` where
+``GraphDSL.create`` returns a :class:`Graph`, in this example a :class:`Graph[ClosedShape, NotUsed]` where
```

Comment:
[This](http://doc.akka.io/docs/akka/2.4/java/stream/stream-quickstart.html#Broadcasting_a_stream) refers to [TwitterStreamQuickstartDocSpec.scala](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-docs/rst/scala/code/docs/stream/TwitterStreamQuickstartDocSpec.scala#L122)

```
RunnableGraph.fromGraph(GraphDSL.create(b -> {
```

where [`GraphDSL.create()`](https://github.com/akka/akka/blob/4a7cd147313404eb8137e56163e54f06c23640d0/akka-stream/src/main/boilerplate/akka/stream/javadsl/GraphCreate.scala.template#L15L16) is defined as follows:

```
  def create[S <: Shape](block: function.Function[GraphDSL.Builder[NotUsed], S]): Graph[S, NotUsed] =
    scaladsl.GraphDSL.create() { b ⇒ block.apply(b.asJava) }
```

So `NotUsed` is correct here.